modules/Microsoft.ManagedIdentity/userAssignedIdentities/deploy.bicep

```
@@ -70,6 +70,9 @@ output resourceId string = userMsi.id
 @description('The principal ID of the user assigned identity.')
```

```suggestion
@description('The principal ID (object ID) of the user assigned identity.')
```

@davidkarlsen @AlexanderSehr 

I would like to suggest that we rename the output descriptions to something that makes it easier to distinguish what each ID is.

---

@ahmadabdalla agree, that was also discussed in issue #3113. Outside of this PR I'd suggest to open an issue to align all module outputs supporting managed identities accordingly

---

adding docs, yes, changing var names no. They should be as in the ARM/bicep API to avoid confusion.

---

opened issue #3139